cmd/cli/serve/serve.go

```diff
@@ -60,6 +60,84 @@ var (
 `))
 )
 
+func GetServerOptions() (*ServeOptions, error) {
```

Comment:
This method generates a `ServerOptions` based on the current configuration, viper handles returning the correct values from the default, file, env vars, and/or flags.